python/lsst/meas/astrom/pessimistic_pattern_matcher_b_3D.py

```diff
@@ -986,9 +1006,14 @@ def _test_spoke(self, src_ctr, src_delta, src_dist, src_ctr_delta,
 
             # Compute the cos between our "center" reference vector and the
             # current reference candidate.
+            proj_ref_delta = \
+                ref_delta_array[ref_dist_idx] - \
+                np.dot(ref_delta_array[ref_dist_idx], ref_ctr) * ref_ctr
```

Comment:
Implied continuation with parentheses would be more consistent with lines 901-906